app/src/views/private/components/comments-sidebar-detail.vue

```diff
@@ -51,46 +53,75 @@ function useActivity(collection: Ref<string>, primaryKey: Ref<PrimaryKey>) {
 	watch([collection, primaryKey], () => refresh());
 
 	return {
-		activity,
-		getActivity,
+		comments,
+		getComments,
 		error,
 		loading,
 		refresh,
-		activityCount,
-		getActivityCount,
+		commentsCount,
+		getCommentsCount,
 		loadingCount,
 		userPreviews,
 	};
 
-	async function getActivity() {
+	async function getComments() {
 		error.value = null;
 		loading.value = true;
 
 		try {
-			const response = await api.get(`/activity`, {
-				params: {
-					'filter[collection][_eq]': collection.value,
-					'filter[item][_eq]': primaryKey.value,
-					'filter[action][_eq]': 'comment',
-					sort: '-id', // directus_activity has auto increment and is therefore in chronological order
-					fields: [
-						'id',
-						'action',
-						'timestamp',
-						'user.id',
-						'user.email',
-						'user.first_name',
-						'user.last_name',
-						'user.avatar.id',
-						'revisions.id',
-						'comment',
-					],
-				},
-			});
+			const [activityResponse, commentsResponse] = await Promise.all([
+				api.get(`/activity`, {
+					params: {
+						'filter[collection][_eq]': collection.value,
+						'filter[item][_eq]': primaryKey.value,
+						'filter[action][_eq]': 'comment',
+						sort: '-id', // directus_activity has auto increment and is therefore in chronological order
+						fields: [
+							'id',
+							'action',
+							'timestamp',
+							'user.id',
+							'user.email',
+							'user.first_name',
+							'user.last_name',
+							'user.avatar.id',
+							'revisions.id',
+							'comment',
+						],
+					},
+				}),
+
+				api.get(`/comments`, {
+					params: {
+						'filter[collection][_eq]': collection.value,
+						'filter[item][_eq]': primaryKey.value,
+						sort: '-date_created',
+						fields: [
+							'id',
+							'comment',
+							'date_created',
+							'user_created.id',
+							'user_created.email',
+							'user_created.first_name',
+							'user_created.last_name',
+							'user_created.avatar.id',
+						],
+					},
+				}),
+			]);
+
+			const mappedActivityComments = activityResponse.data.data.map((activity: Activity) => ({
+				id: activity.id,
+				comment: activity.comment,
+				date_created: activity.timestamp,
+				user_created: activity.user,
+			}));
 
-			userPreviews.value = await loadUserPreviews(response.data.data, regex);
+			const response = [...mappedActivityComments, ...commentsResponse.data.data];
 
-			const activityWithUsersInComments = (response.data.data as Activity[]).map((comment) => {
+			userPreviews.value = await loadUserPreviews(response, regex);
```

Comment:
Passing in an array now changes the function signature, the type is now wrong for `loadUserPreviews` see line 246

Reply:
Great eye, it looks like an error carried forward, where `Activity[]` was passed back then 👍 Updated in https://github.com/directus/directus/pull/22295/commits/02c1be310978c99cb5c056c74363c8474cf55563